src/ome_zarr_models/v04/bioformats2raw.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any, Literal
+
+from pydantic import Field
+
+from ome_zarr_models.base import Base
+from ome_zarr_models.v04.plate import Plate
+
+
+class BioFormats2Raw(Base):
+    """
+    A bioformats2raw zarr group.
```

Comment:
does this model a zarr group, or the attributes inside that group?

Reply:
Just the attributes inside that group - should I rename to `BioFormats2RawAttributes` then?

Reply:
maybe `BioFormats2RawAttrs`, in keeping with our use of the same abbreviation over in multiscales.py